confidant/schema/credentials.py

```diff
@@ -21,6 +21,7 @@ class CredentialResponse(object):
     permissions = attr.ib(default=dict)
     tags = attr.ib(default=list)
     last_rotation_date = attr.ib(default=None)
```

Comment:
not related to this PR but we could remove last_rotation_date if we aren't using it (which I don't think we should given that next_rotation_date has some complex logic)

If you remove it remove the line in the test too

Reply:
I'm ok with delivering it, even if it's not shown in the UI. We may decide to show it later, or folks may want to use it in other tooling